.pre-commit-config.yaml

```diff
@@ -90,12 +90,12 @@ repos:
           - --env
           - LOG_LEVEL=warning
           - --release
-          - v5.0.7
+          - v5.10.0
```

Comment:
similar to the last comment  - this update is not referenced within the commit message

Reply:
The commit subject is "chore(MegaLinter): Upgrade from v5.0.7 to v5.10.0."

Reply:
comment redacted